Lobe_iOS.xcodeproj/project.pbxproj

```diff
@@ -438,18 +438,18 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"Lobe_iOS/Preview Content\"";
-				DEVELOPMENT_TEAM = UBF8T346G9;
+				DEVELOPMENT_TEAM = TRXLK9K735;
 				ENABLE_PREVIEWS = YES;
 				INFOPLIST_FILE = Lobe_iOS/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.lobe;
+				PRODUCT_BUNDLE_IDENTIFIER = com.bootstrap.lobe;
```

Comment:
Will need to revert this.

Reply:
(And update docs)

Reply:
Passing along my investigation here:
- A software simulator can run the application with or without a dev account.
- However, downloading app to device requires:
    - Having an Apple developer account.
    - Creating a unique bundle ID so that you can create provisioning profiles, used 
- The bundle ID we use is set up with Microsoft’s developer account, which is useful for the team for provisioning devices under the same account. This means that Lobe engineers won’t need to purchase their own Apple Developer account to contribute. Customers will have to change the bundle ID no matter what.

To-dos are to revert the bundle ID and development team as well as update README with some of the above info.